src/LocationManager.swift

```diff
@@ -34,23 +48,30 @@ open class LocationManager: NSObject, CLLocationManagerDelegate {
     coreLocationManager.delegate = self
   }
 
+  /// Request the user give the application location access at all times.
   open func requestAlwaysAuthorization() {
     coreLocationManager.requestAlwaysAuthorization()
-
   }
 
+  /// Request the user give the application location access only when in the foreground.
   open func requestWhenInUseAuthorization() {
     coreLocationManager.requestWhenInUseAuthorization()
   }
 
+  /// Asks the location subsystem if we're currently authorized for location access while in foreground.
+  /// - returns: Whether or not the application is authorized.
   open func isInUseAuthorized() -> Bool {
     return CLLocationManager.authorizationStatus() == .authorizedWhenInUse ? true :  false
```

Comment:
is `? true :  false` needed here or can the return value just be the result of the comparison?

```swift
return CLLocationManager.authorizationStatus() == .authorizedWhenInUse
```